header/header.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
There is std `errors` pkg, we don't need to import anything.

Reply:
Specifically for wrapping functionality `fmt.Errorf(%w, err)` will do a trick.

Reply:
Made the change, note that now, the formatted error message with the validator set hashes etc. is not propagated.

Reply:
@Manav-Aggarwal, wait, %w is just an example, you do this:

`fmt.Errorf("%w: expected old header next validators (%X) to match those from new header (%X)", err, values...)`

Reply:
Thanks for clarifying, made the change.